### PR TITLE
Data protection context

### PR DESCRIPTION
**Pull request by williamhogman**

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -167,24 +167,38 @@ unsecured loan customers.
...
 | extensions      | 0..1 | ExtensionPoint | v0 |                                     |
 
+DataProtectionContext
+
+| String value | Remark                                                   |
+|--------------|----------------------------------------------------------|
+| REAL         | Production data consisting concerning real data-subjects |
+| FICTIONAL    | Fictional data, does not concern real data-subjects      |
+
+Consumers MAY reject or refuse to process data sent in the `REAL` data
```
Comment (zinob) [inline spec.md]: While Consumer is technically correct i would preffer the wording receiver or recipient due to the ambiguous meaning of the word consumer in this context

File: spec.md [hunk 2]
```diff
@@ -293,3 +307,71 @@ MaritalStatus
...
+## PrivateUnsecuredLoanRejection
+
+This event pertains denials of unsecured loans for Swedish customers.
+
+| Name            | C.   | Type        | V. | Remark                          |
+|-----------------|------|-------------|----|---------------------------------|
+| rejectionReason | 0..1 | Application | v0 | Contains the loan application   |
+| broker          | 1    | reverse-dns | v0 | Domain-name of the broker       |
+| brokerReference | 1    | string      | v0 | Per-broker unique, not globally |
+| Cleanex         | 1    |             | v0 |                                 |
```
Comment (zinob) [inline spec.md]: Please delete this <3

File: spec.md (hunk 1 above)
Comment (williamhogman) [reply spec.md]:
```
The standard defines the behaviour of two actors, the producer and the
consumer, as defined in the CloudEvents specification```
Rhee

Comment (williamhogman) [reply spec.md]: I agree on this point though.

Comment (williamhogman) [reply spec.md]: #22

Comment (zinob) [reply spec.md]: I know that it is defined in the standard, but sometime the standard is not perfectly applicable to every use-case.

Comment (williamhogman) [reply spec.md]: Agree but moving this to #22

Comment (williamhogman) [reply spec.md]: Given that this is done in #22 can we merge?

**Pull request merged**